openstack/nova/templates/api-metadata-deployment.yaml

```diff
@@ -47,7 +47,7 @@ spec:
             - kubernetes-entrypoint
           env:
             - name: COMMAND
-              value: "nova-api"
+              value: "nova-api-metadata"
```

Comment:
We can't do that. From the [docs](https://docs.openstack.org/nova/latest/admin/cells.html#cells-v2-layout-metadata-api):
```
Global

If you have networks that span cells, you might need to run Nova metadata API globally. When running globally, it should be configured as an API-level service with access to the [api_database.connection](https://docs.openstack.org/nova/latest/configuration/config.html#api_database.connection) information. The nova metadata API service must not be run as a standalone service, using the nova-api-metadata service, in this case.
```

Reply:
Changed it back